server/app/services/program/ProgramService.java

```diff
@@ -635,6 +676,13 @@ private ImmutableSet<CiviFormError> validateProgramData(
     if (displayMode.isBlank()) {
       errorsBuilder.add(CiviFormError.of(MISSING_DISPLAY_MODE_MSG));
     }
+    var validNotificationPrefereces =
```

Comment:
**nitpick (non-blocking):** I'm not sure if we have a policy on var, but I don't like it and I think it adds cognitive load to reading this chunk of code.